src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,168 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
+facts about these functions.
+
+These results are useful for proving that the uniform limit of a sequence of holomorphic functions
+is holomorphic.
+
+-/
+
+open set measure_theory metric filter function
+open_locale interval real
+
+noncomputable theory
+
+variables {E : Type} [normed_group E] [normed_space ℂ E]
+
+namespace complex
+
+/-- Given a function `f : ℂ → E`, this gives the function $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x`
+runs over a circle of radius `R` around `z`. If `f` is differentiable and `w` is in the interior of
+the ball, then the integral from `0` to `2 * π` of this gives the value `f(w)`. -/
+def circle_integral_transform (R : ℝ) (z w : ℂ) (f : ℂ → E) : ℝ → E :=
+  λ θ, (2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ) - w)⁻¹ • f (circle_map z R θ)
+
+/-- The derivative of `circle_integral_transform` w.r.t `w` -/
+def circle_integral_transform_deriv (R : ℝ) (z w : ℂ) (f : ℂ → E) (θ : ℝ) : E :=
+  (2 * ↑π * I)⁻¹ • deriv (circle_map z R) θ • ((circle_map z R θ - w) ^ 2)⁻¹ • f (circle_map z R θ)
+
+/-- Cauchy integral form of a function around a disk of radius `R` around `z` -/
+def circle_integral_form [complete_space E] (R : ℝ) (z : ℂ) (f : ℂ → E) : ℂ → E :=
+  λ w, (2 * ↑π * I)⁻¹ • (∮ z in C(z, R), (z - w)⁻¹ • f z)
+
+lemma circle_integral_form_eq_int [complete_space E] (R : ℝ) (z : ℂ) (f : ℂ → E) :
+  circle_integral_form R z f = λ w,
+ ∫ (θ : ℝ) in 0..2 * π, (circle_integral_transform R z w f ) θ :=
+begin
+  simp_rw [circle_integral_form, circle_integral_transform, circle_integral,
+    interval_integral.integral_smul],
+end
+
```

Comment:
I don't see this definition (and its lemma) used anywhere. Are you sure we want them?
```suggestion
```

Reply:
I note that @loefflerd highlights these are used in #13500. I suggest we leave them out of this PR since there's already enough going on.